filesystems/s3/src/main/java/org/lerch/s3fs/S3FileSystemProvider.java

```diff
@@ -1,17 +1,7 @@
 package org.lerch.s3fs;
 
-import software.amazon.awssdk.core.sync.RequestBody;
-import software.amazon.awssdk.core.ResponseInputStream;
-import software.amazon.awssdk.services.s3.S3Client;
-import software.amazon.awssdk.services.s3.model.Bucket;
-import software.amazon.awssdk.services.s3.model.CopyObjectRequest;
-import software.amazon.awssdk.services.s3.model.CreateBucketRequest;
-import software.amazon.awssdk.services.s3.model.DeleteObjectRequest;
-import software.amazon.awssdk.services.s3.model.GetObjectAclRequest;
-import software.amazon.awssdk.services.s3.model.GetObjectRequest;
-import software.amazon.awssdk.services.s3.model.GetObjectResponse;
-import software.amazon.awssdk.services.s3.model.PutObjectRequest;
-import software.amazon.awssdk.services.s3.model.S3Exception;
+import com.google.cloud.Tuple;
```

Comment:
I guess it's ok as long as it works, but still feels a bit strange to use `com.google.cloud.Tuple` to do AWS stuff. Is it possible to use a `Pair` from Apache Commons instead?

Reply:
Agreed. I decided to use an Apache commons `ImmutablePair` as it is in a lambda and doesn't need to mutate